kolibri/auth/permissions/general.py

```diff
@@ -167,6 +170,9 @@ def user_can_delete_object(self, user, obj):
     def readable_by_user_filter(self, user, queryset):
         # TODO(jamalex): reimplement this method in a more efficient way
 
+        # import here to avoid circular imports
+        from ..models import Collection
```

Comment:
Is there any way to avoid imports like this? Obviously these classes could live in `..models` too, but it might become a very large module. In your opinion, is it cleaner to e.g. move `Collection` and it's permissions into another module, e.g. `..collections` and in the `models` module include a line like:
```python
from collections import Collection
```